src/enochecker/checkerservice.py

```diff
@@ -25,17 +25,17 @@
 Optional = collections.namedtuple("Optional", "key type default")
 Required = collections.namedtuple("Required", "key type")
 
-CHECKER_METHODS = [
+CHECKER_METHODS: List[str] = [
     "putflag",
     "getflag",
     "putnoise",
     "getnoise",
     "havoc",
     "exploit",
-]  # type: List[str]
+]
 
 # The json spec a checker request follows.
-spec = [
+spec: List[Union[Required, Optional]] = [
```

Comment:
that also breaks in older python3, right?

Reply:
As mentioned in #33 we basically require 3.6 or aboe anyways